suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -37,25 +49,35 @@ async function fetchDIDLists(I) {
     ).then((res) => new Gen3Response(res));
 
     const projectAccessList = httpResp.body.project_access;
-
+    
     // initialize dict of accessible DIDs
     let ok200files = {}; // eslint-disable-line prefer-const
     // initialize dict of blocked DIDs
     let unauthorized401files = {}; // eslint-disable-line prefer-const
 
-    // adding record DIDs to their corresponding ACL key
-    // ( I.cache.records is created in BeforeSuite() )
-    I.cache.records.forEach((record) => {
-      // console.log('ACLs for ' + record['did'] + ' - ' + record['acl']);
-      // Filtering accessible DIDs by checking if the record acl is in the project access list
-      const accessibleDid = record.acl.filter(
-        (acl) => projectAccessList.hasOwnProperty(acl) || record.acl === '*', // eslint-disable-line no-prototype-builtins
-      );
+    // assemble 200 authorized ids
+    const authorized200Project = ['*'];
+    for (const i in projectAccessList) { // eslint-disable-line guard-for-in
```

Comment:
this would go away too

Reply:
this one would get me the projects I am having access too, and adding them to the list of projects which would be used later to get the guids associated with these projects